org.gnome.Logs.json

```diff
@@ -0,0 +1,117 @@
+{
+  "app-id": "org.gnome.Logs",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.32",
+  "sdk": "org.gnome.Sdk",
+  "command": "gnome-logs",
+  "finish-args": [
+    "--share=ipc", "--socket=x11",
+    "--socket=wayland",
+    "--filesystem=/var/log/journal:ro",
```

Comment:
Does it actually work? Have you tested it? Are you sure, that these logs come from the host, not the sandbox?
http://docs.flatpak.org/en/latest/sandbox-permissions.html#filesystem-access

Reply:
Yes, I've tested it. It shows logs from the host.